Example/DApp/Sign/Connect/ConnectViewController.swift

```diff
@@ -63,7 +63,8 @@ class ConnectViewController: UIViewController, UITableViewDataSource, UITableVie
     }
 
     @objc func connectWithExampleWallet() {
-        let url = URL(string: "https://walletconnect.com/wc?uri=\(uri.absoluteString)")!
+		let encodedUri = uri.absoluteString.addingPercentEncoding(withAllowedCharacters: .rfc3986)!
```

Comment:
`WalletConnectURI` object already has a convenience computed property called`deeplinkUri` so you can use that.